service/worker/scanner/executions/workflows.go

```diff
@@ -35,13 +35,13 @@ import (
 
 const (
 	// ConcreteScannerContextKey is the key used to access ScannerContext in activities for concrete executions
-	ConcreteScannerContextKey = ContextKey(0)
```

Comment:
Using string instead of number will make it much unlikely to conflict. The performance penalty should be minimal enough to ignore.